services/sorting-room/makeConfig.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+ENV_FILE_LOCATION="dist/assets/js/env.js"
+rm -f $ENV_FILE_LOCATION;
+
+/bin/cat > $ENV_FILE_LOCATION <<EOF
```

Comment:
Do you get PATH errors without the `/bin/` prefix? Just `cat` would be better to handle systems that don't symlink /bin to /usr/bin.

Reply:
I don't recall, I think I may have had problems when using the alpine image.

Reply:
`/usr/bin/cat` won't work on Alpine, but just `cat` will.

Reply:
```
> $ docker run -i -t alpine:latest sh                                                                                  
Unable to find image 'alpine:latest' locally                                                                           
latest: Pulling from library/alpine                                                                                    
c9b1b535fdd9: Pull complete                                                                                            
Digest: sha256:ab00606a42621fb68f2ed6ad3c88be54397f981a7b70a79db3d1172b11c4367d
Status: Downloaded newer image for alpine:latest                                                                       
/ # echo $PATH                                                                                                         
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin                    
/ # which cat                                                                                                          
/bin/cat     
```